packages/composite-checkout/src/components/radio-button.js

```diff
@@ -100,11 +100,32 @@ const RadioButtonWrapper = styled.div`
 	:hover svg {
 		filter: grayscale( 0 );
 	}
+
+	${handleWrapperDisabled};
```

Comment:
Just noting that I copied these styles over from here:

![image](https://user-images.githubusercontent.com/6981253/83175356-c1f36780-a0e9-11ea-89ab-01ee27cd57e8.png)

I'm wondering whether it makes sense to have two versions of this file?

Reply:
For now, I think yes, because they're in two separate packages. We should eventually move them to one file in a third package that both can import, like `@automattic/components` or something like that.